src/js/components/DataTableColumns/DataTableColumns.js

```diff
@@ -164,13 +165,18 @@ export const DataTableColumns = ({ drop, options, ...rest }) => {
   const { format } = useContext(MessageContext);
   const theme = useContext(ThemeContext);
   const [showContent, setShowContent] = useState();
+  const contextValue = useMemo(() => ({ inDataColumn: true }), []);
```

Comment:
```suggestion
  const contextValue = useMemo(() => ({ inDataTableColumns: true }), []);
```
Can we call it this to align with component name?

Reply:
done